<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>844</width>
          <height>509</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="gun_conf_tab">
            <attribute name="title">
             <string>Конфигуратор пушек</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="7" column="2" rowspan="2">
              <widget class="QGroupBox" name="player_param_group_box">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="title">
                <string>параметры игрока с этим оружеем</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="2" column="1">
                 <widget class="QCheckBox" name="gst_friendlyfire_check_box">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>количество жизней(1-250):</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>получать урон от своих:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="gst_player_live_count_spin_box">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>999</number>
                  </property>
                  <property name="singleStep">
                   <number>5</number>
                  </property>
                  <property name="value">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="gst_stun_time_label">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>время шока 0 сек:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="gst_stun_time_slider">
                  <property name="maximum">
                   <number>23</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QGroupBox" name="gun_name_group_box">
               <property name="title">
                <string>название конфигурации и модели оружия</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="2" column="1">
                 <widget class="QLineEdit" name="gst_gun_model_line_edit">
                  <property name="locale">
                   <locale language="English" country="UnitedStates"/>
                  </property>
                  <property name="inputMask">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string notr="true"/>
                  </property>
                  <property name="maxLength">
                   <number>32</number>
                  </property>
                  <property name="dragEnabled">
                   <bool>false</bool>
                  </property>
                  <property name="placeholderText">
                   <string>например: AK-47</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="gst_gun_name_line_edit">
                  <property name="locale">
                   <locale language="English" country="UnitedStates"/>
                  </property>
                  <property name="inputMask">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string notr="true"/>
                  </property>
                  <property name="maxLength">
                   <number>32</number>
                  </property>
                  <property name="dragEnabled">
                   <bool>false</bool>
                  </property>
                  <property name="placeholderText">
                   <string>например: ak47_for_red_team</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>имя конфигурации:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>модель оружия:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="6" column="2">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="1" rowspan="9">
              <widget class="QGroupBox" name="groupBox_7">
               <property name="title">
                <string>список конфигураций</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="2" column="0">
                 <widget class="QPushButton" name="add_gun_config_button">
                  <property name="text">
                   <string>Добавить конфигурацию</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QListWidget" name="gun_config_list_widget">
                  <item>
                   <property name="text">
                    <string>AK-47</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>AWP</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="remove_gun_config_button">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>Удалить конфигурацию</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="10" column="1" colspan="2">
              <widget class="QGroupBox" name="gun_param_group_box">
               <property name="title">
                <string>параметры оружия</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="5" column="1">
                 <widget class="QSlider" name="gst_gun_fire_temp_slider">
                  <property name="maximum">
                   <number>11</number>
                  </property>
                  <property name="pageStep">
                   <number>1</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="5" column="2">
                 <widget class="QLabel" name="label">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>урон:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>перезарядка:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>тип стрельбы:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="10" column="3">
                 <widget class="QSpinBox" name="gst_gun_clip_size_spin_box">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>30</number>
                  </property>
                 </widget>
                </item>
                <item row="10" column="2">
                 <widget class="QLabel" name="ammo_in_mag_count_lable">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>размер обоймы(1-200):</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="8" column="2">
                 <widget class="QLabel" name="reload_time_lable">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>скорость перезарядки(сек):</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QComboBox" name="gst_gun_reload_type_combo_box">
                  <item>
                   <property name="text">
                    <string extracomment="поменять магазин и взвести затвор">смена магазин</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string extracomment="зарядка по одному патрону, как у дробовиков">помповая дозорядка</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>перезарядка по нажатию кнопки</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="10" column="0">
                 <widget class="QLabel" name="magazine_count_lable">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>количество обойм(0-100):</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="gst_gun_fire_temp_label">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>темп огня: 250 выст/мин:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="10" column="1">
                 <widget class="QSpinBox" name="gst_gun_clip_count_spin_box">
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>4</number>
                  </property>
                 </widget>
                </item>
                <item row="5" column="3">
                 <widget class="QComboBox" name="gst_gun_damage_combo_box">
                  <property name="currentText">
                   <string>1</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>4</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>15</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>17</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>20</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>25</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>30</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>35</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>40</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>50</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>75</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>100</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="8" column="3">
                 <widget class="QSpinBox" name="gst_gun_reload_time_spin_box">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>20</number>
                  </property>
                  <property name="singleStep">
                   <number>100</number>
                  </property>
                  <property name="value">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1" colspan="2">
                 <widget class="QComboBox" name="gst_gun_burst_size_combo_box">
                  <item>
                   <property name="text">
                    <string>ручное досылание патрона</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>одиночные выстрелы</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>с отсечкой по 2 патрона</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>с отсечкой по 3 патрона</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>с отсечкой по 4 патрона</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>с отсечкой по 5 патронов</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>с отсечкой по 6 патронов</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>автоматический огонь</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="add_gun_tab">
            <attribute name="title">
             <string>Список пушек</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_10">
               <property name="title">
                <string>Список оружия</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QTableView" name="inventary_gun_table"/>
                </item>
                <item>
                 <widget class="QPushButton" name="load_config_on_gun_button">
                  <property name="text">
                   <string>Назначить конфигурацию</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
